proto.md

```diff
@@ -194,7 +193,8 @@ Grundstücke
 		S: -NEIN 131 insufficient money, need <amount>
 		S: -NEIN 136 belongs to player <player>
 		S: -NEIN you are not a player
-		S: add-money <Preis> Buy plot <Grundstück>
+		S: show-transaction <Preis> Buy plot <Grundstück>
+		S: money-update ...
```

Comment:
`add-money` was renamed because its name didn't fit anymore.
`money-update` has stayed the same. but I didn't want to write out the args, so I wrote `...`